roles/openldap/defaults/main.yml

```diff
@@ -0,0 +1,52 @@
+keystone:
```

Comment:
Yes agree, can we simply move it one level higher. So <b>keystone.ldap.enabled</b> becomes <b>ldap.enabled</b>, want to confirm before committing as there are a lot of references to the variables.

  ldap:
    enabled: False
    url: ldap://0.0.0.0:389
    dc: "example"
    suffix: "dc=example,dc=org"
    root_cn: "admin"
    root_dn: "cn=admin,dc=example,dc=org"
    company_ou: "openstack"
    organizationalUnit: "ou=openstack,dc=example,dc=org"
    users_ou: "users"
    groups_ou: "groups"
    users_dn: "ou=users,ou=openstack,dc=example,dc=org"
    groups_dn: "ou=groups,ou=openstack,dc=example,dc=org"
    password_policy_enabled: False
    password_policy_ou: "policies"